scripts/staging/slicing/base/init.py

```diff
@@ -0,0 +1,45 @@
+import pandas as pd
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import OneHotEncoder
+
+
+def init():
+    file_name = 'salary.csv'
```

Comment:
It would be good to refactor this code (into functions and classes) such that we can process any input datasets (e.g., passed as a dataframe). Accordingly, I would recommend to remove hard-coded file names and file I/O from the actual preparation and slicing code.

Reply:
In "tests" folder I added the example of current setup for slicer processing which is now gets constants and prepared initial dataset via arguments.